### PR TITLE
Update checks config docs and readme, fix import error in utilities r…

### PR DESCRIPTION
**Pull request by DanielHHowell**
…egex parsing

### CODE REVIEW

File: docs/checks.md [hunk 1]
```diff
@@ -304,80 +304,92 @@ class HaltOnFailed(str, enum.Enum):
...
 ```console
-CONNECTOR    STATUS    ERRORS
-servo        X FAILED  (1/1) Opsani API connectivity: Response status code: 500
-prometheus   √ PASSED
-vegeta       √ PASSED
+CONNECTOR                        STATUS    ERRORS
+opsani_dev                       √ PASSED
+opsani-dev:kubernetes            √ PASSED
+opsani-dev:prometheus            √ PASSED
+opsani-dev:kube-metrics          √ PASSED
```
Comment (linkous8) [inline docs/checks.md:363]:
```suggestion
CONNECTOR                        STATUS    ERRORS
test.optimizer.com/test          X FAILED  (1/1) Opsani API connectivity: ['Response status code: 404']
opsani_dev                       √ PASSED
opsani-dev:kubernetes            √ PASSED
opsani-dev:prometheus            X FAILED  (1/1) Connect to "http://localhost:9090": ['caught exception (ConnectError): [Errno 61] Connection refused']
opsani-dev:kube-metrics          √ PASSED
```

Add errors to increase documentational coverage

File: docs/checks.md [hunk 2]
```diff
@@ -304,80 +304,92 @@ class HaltOnFailed(str, enum.Enum):
...
+```console
+    checks:
+      check_halting: True
 ```
 
-Results get aggregated and summarized by connector:
+Results from checks can be outputted into a table
+```console
+    checks:
+      progressive: True
```
Comment (linkous8) [inline docs/checks.md:355]:
```suggestion
      progressive: False
```

This seems to be a misnomer that I wish I'd caught sooner but its too late to fix it now. I'll make a mental note to ensure its not carried over in the refactor 🙃 (also it has to be false to get the table)

**Review by linkous8 — CHANGES_REQUESTED** [review]
https://github.com/opsani/servox/blob/2a21c5e58568e868845664398b1d5146d4d9538f/README.md?plain=1#L98

Update to `servo run --dry-run`

We can also get rid of the Open Questions section now 🙂 https://github.com/opsani/servox/blob/2a21c5e58568e868845664398b1d5146d4d9538f/docs/checks.md?plain=1#L480-L505

File: docs/checks.md [hunk 3]
```diff
@@ -304,80 +304,92 @@ class HaltOnFailed(str, enum.Enum):
...
+      container: frontend
+      service: frontend
+
+      envoy_sidecar_image: opsani/envoy-proxy:v1.20.1post1
+      cpu:
+        min: 250m
+        max: '3.0'
+      memory:
+        min: 128.0MiB
+        max: 3.0GiB
```
[the remark this answers is not in the document]
Comment (DanielHHowell) [reply docs/checks.md:329]: I thought about just including the checks config and do so in the examples below, but I thought a 'complete' example would be good to see at the top to visually grasp. I can add a line explaining accordingly, or if you feel strongly about it remove the 'complete' sample yaml per suggestion

File: docs/checks.md [hunk 4]
```diff
@@ -304,80 +304,81 @@ class HaltOnFailed(str, enum.Enum):
...
-Options:
-  [CONNECTORS]...                 Connectors to check
-  -n, --name TEXT                 Filter by name  [default: False]
-  -i, --id TEXT                   Filter by ID  [default: False]
-  -t, --tag TEXT                  Filter by tag  [default: False]
-  -h, --halt-on-failed [requirement|check|never]
-                                  Halt running checks on a failure condition
-                                  [default: requirement]
+```servo.yaml
+    ...
```
Comment (linkous8) [inline docs/checks.md:317]:
```suggestion
    opsani_dev:
      ...
```

**Review by linkous8 — APPROVED** [review]
Looks good, thanks again for getting this documented 📝 🎉

**Pull request merged**